docs/collecting-data/collecting-from-own-applications/snowplow-tracking-cli/installation/index.md

```diff
@@ -4,8 +4,11 @@ date: "2020-10-12"
 sidebar_position: 100
 ---
 
-You can download the binary for Linux and Windows directly from Github:
+You can download the binary for Linux, macOS and Windows directly from GitHub:
 
-- [**Linux 64bit Binary**](https://github.com/snowplow/snowplow-tracking-cli/releases/download/0.4.0/snowplow_tracking_cli_0.4.0_linux_amd64.zip) (SHA1 checksum: 30dfc7ad6e5ad7c17d724c1dcfec4e21172382b6)
-- [**Windows 64bit**](https://github.com/snowplow/snowplow-tracking-cli/releases/download/0.4.0/snowplow_tracking_cli_0.4.0_windows_amd64.zip) **[Binary](https://github.com/snowplow/snowplow-tracking-cli/releases/download/0.4.0/snowplow_tracking_cli_0.4.0_windows_amd64.zip)** (SHA1 checksum: cd4ebd5aeb4b6d156287941dac171097f34355ce)
-- [**macOS 64bit**](https://github.com/snowplow/snowplow-tracking-cli/releases/download/0.4.0/snowplow_tracking_cli_0.4.0_darwin_amd64.zip) **[Binary](http://snowplow_tracking_cli_0.4.0_darwin_amd64.zip)** (SHA1 checksum: 41e477866f4ed697d1b781c4027276d9b6e63664)
+* [**Linux AMD 64bit Binary**](https://github.com/snowplow/snowplow-tracking-cli/releases/download/0.7.0/snowplow_tracking_cli_0.7.0_linux_amd64.zip)
```

Comment:
P.S. Bonus points if you can make this use `src/componentVersions.js` instead :)

Reply:
Sorry saw this too late!